smileidentity/android
# feat: jobsubmission abstraction

Story: https://app.shortcut.com/smileid/story/13730/improve-error-handling-on-android-and-ios-sdk
Improve Error Handling on android and ios sdk.
## Summary

Better abstraction for 
1. Smartselfie enrollment
2. Smartselfie authentication
3. Document verification
4. Enhanced document verification
5. Biometric KYC

## Known Issues

n/a

## Test Instructions

1. Test the jobs from the orchestrated screens paths
2. Test offline mode 
3. Capture jobs and then use the submitJob method to make sure everything is working fine 

## Screenshot

n/a
